app/frontend/controllers/notes_controller.js

```diff
@@ -18,8 +18,10 @@
 
 let CodeMirror = null;
 
+const { set } = require('electron-settings');
```

Comment:
Has this been included accidentally? All settings should be handled via ipcSettings.

Reply:
Yes, that was an accident